load_data.py

```diff
@@ -118,7 +118,7 @@ def load_article(article):
     print 'loading article...'
 
 def load_schema_dir(dirpath):
-    for schema_file in os.listdir(dirpath):
+    for schema_file in sorted(os.listdir(dirpath)):
```

Comment:
It would be nice to limit to *.txt as then a README.md or similar could be dropped into the directory without interfering with data loading. I can add the glob.glob('*.txt'), but that is a nice-to-have and the current patch is 'must-have'.